flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/plan/nodes/physical/batch/BatchExecSortWindowAggregate.scala

```diff
@@ -96,8 +96,4 @@ class BatchExecSortWindowAggregate(
     val aggregateNamePrefix = if (isMerge) "Global" else "Complete"
     aggregateNamePrefix + "WindowSortAggregateBatchExec"
   }
-
-  override def getParallelism(input: StreamTransformation[BaseRow], conf: TableConfig): Int = {
-    if (isFinal && grouping.length == 0) 1 else input.getParallelism
```

Comment:
How do you set parallelism 1 in this case? Maybe you can add some comments in `BatchShuffleStageParallelismCalculator`?

Reply:
> How do you set parallelism 1 in this case? Maybe you can add some comments in `BatchShuffleStageParallelismCalculator`?

I add comments in BatchFinalParallelismSetter, as its parallelism is set to 1 depends on the global exchange.